dracut/30ignition/module-setup.sh

```diff
@@ -53,6 +56,7 @@ install() {
     install_ignition_unit ignition-mount.service
     install_ignition_unit ignition-files.service
     install_ignition_unit ignition-remount-sysroot.service
+    install_ignition_unit ignition-reboot.service
```

Comment:
Maybe let's call it `ignition-apply-kargs.service` or something?

Reply:
I'd prefer to drop `ignition` from the name entirely. It's not running Ignition at all. It would be good to keep `reboot` in the unit name as well to make it clear what it's doing. Just spitballing `apply-kargs-and-reboot.service`?

Reply:
I'm not a fan of unit names that don't contain any hint of where it came from. The `ignition` in the unit name is more to namespace it in that sense (maybe `ignition-dracut-...` would work too). Of course, we could (and probably should) also add e.g. `Documentation=https://github.com/coreos/ignition-dracut` or something, which I don't think we do today.

Reply:
Changed the name to `ignition-apply-kargs.service` and `ignition-apply-kargs.sh`, and added `Reboot after Ignition to apply kargs` to description.